development/build/scripts.js

```diff
@@ -635,12 +516,16 @@ function createFactoredBuild({
     const environment = getEnvironment({ buildTarget });
     const config = await getConfig(buildType, environment);
     const { variables, activeBuild } = config;
-    await setEnvironmentVariables({
-      buildTarget,
+    setEnvironmentVariables({
+      isDevBuild: reloadOnChange,
+      isTestBuild: isTestBuild(buildTarget),
+      buildName: getBuildName({
+        environment,
+        buildType,
+      }),
       buildType,
       environment,
       variables,
-      activeBuild,
```

Comment:
`activeBuild` was unused. It _was_ being passed around to other functions within `setEnvironmentVariables`, but those functions didn't actually use the value. So I've removed it.